.ci/build-mac.sh

```diff
@@ -54,14 +74,16 @@ mkdir build && cd build || exit 1
     -DLLVM_INCLUDE_UTILS=OFF -DLLVM_USE_PERF=OFF -DLLVM_ENABLE_Z3_SOLVER=OFF \
     -DUSE_NATIVE_INSTRUCTIONS=OFF \
     -DUSE_SYSTEM_MVK=OFF \
+    -DUSE_SYSTEM_FAUDIO=ON \
+    -DUSE_SYSTEM_SDL=ON \
     $CMAKE_EXTRA_OPTS \
     -DLLVM_TARGET_ARCH=X86_64 -DCMAKE_OSX_ARCHITECTURES=x86_64 -DCMAKE_IGNORE_PATH="$BREW_PATH/lib" \
     -G Ninja
 
 "$BREW_PATH/bin/ninja"; build_status=$?;
 
 cd ..
-
+export BUILDING_FOR="X64"
```

Comment:
I don't understand why this is randomly shoved in this line, making the file less readable yet again.

Reply:
Future proofing as this was taken from my arm ci pr

Reply:
Emphasis on "this line" and "readable"

Reply:
What would you like me to do to make it more readable